base.Dockerfile

```diff
@@ -3,7 +3,7 @@
 FROM openshift/origin-release:golang-1.13
 
 # Install test dependencies
-RUN yum install -y skopeo && \
+RUN yum install -y skopeo gpgme-devel && \
     export OS=$(go env GOOS) && \
```

Comment:
I think we discovered that this change would need to go into master before the changes take effect, fyi